scalyr_agent/tests/monitors_manager_test.py

```diff
@@ -39,7 +39,32 @@ def test_single_module(self):
         self.assertEquals(len(test_manager.monitors), 1)
         self.assertEquals(test_manager.monitors[0].monitor_name, 'test_monitor()')
 
-    def test_multiple_modules(self):
+    def test_multiple_modules_and_platform_monitors(self):
```

Comment:
I'm not sure what you mean by `platform_monitors` here.  It seems like you are just testing multiple monitors.  Also, why was it necessary to change from 2 to 3?